Classes/ios/UIViewController+MBDebugPanel.m

```diff
@@ -0,0 +1,157 @@
+//
+//  UIViewController+MBDebugPanel.m
+//  MBDebugPanel
+//
+//  Created by Ian on 3/27/15.
+//
+//
+
+#import "UIViewController+MBDebugPanel.h"
+
+#import "MBDebugPanel.h"
+
+#import <JGMethodSwizzler/JGMethodSwizzler.h>
+
+#import <objc/runtime.h>
+
+NSString *const MB_UIViewController_Utility_debugMenuItemsManagementEnabledKey = @"MB_UIViewController_Utility_debugMenuItemsManagementEnabled";
+NSString *const MB_UIViewController_Utility_managedDebugMenuItemsKey = @"MB_UIViewController_Utility_managedDebugMenuItemsKey";
```

Comment:
These keys can be simple `char *` pointers, which would eliminate the need for the `CFBridingRetain` when setting/getting the associated object.  In fact, associated object storage/lookup is done via strict pointer comparison, so there's actually no need to set a value.   (I could be missing a reason you chose this approach deliberately.. in which case.. disregard.)

```objc
const char * MB_UIViewController_Utility_debugMenuItemsManagementEnabledKey;
const char * MB_UIViewController_Utility_managedDebugMenuItemsKey;
```